include/service/entities/plugin_service.js

```diff
@@ -36,7 +37,14 @@ module.exports = function PluginServiceModule(pb) {
      * @module Services
      * @submodule Entities
      */
-    function PluginService(){
+    function PluginService(siteUID){
```

Comment:
The plugin service is going to have to change over to extend BaseObjectService at some point soon.  If we are going to change the constructor here I'd like to go ahead and make it consistent with the other services by passing an options object.